src/main/java/core/selenium/WebElementAction.java

```diff
@@ -172,11 +171,47 @@ public String getSiteCurrentUrl() {
      * Clicks the checkbox elements.
      *
      * @param webElement checkbox.
-     * @param isPrivate set the status from private.
+     * @param stateOfCheckbox true to select, false to deselect.
      */
-    public void clickCheckBox(final WebElement webElement, final boolean isPrivate) {
-        if (!webElement.isSelected() && isPrivate) {
+    public void selectCheckBox(final WebElement webElement, final boolean stateOfCheckbox) {
+        if (!webElement.isSelected() && stateOfCheckbox) {
+            webElement.click();
+        }
+        if (webElement.isSelected() && !stateOfCheckbox) {
             webElement.click();
         }
     }
+
+    /**
+     * Gets the generic text from the header in a created opportunity page.
+     * @param xpath initial Xpath of the element.
```

Comment:
You should add a newLine to match the checkstyle convention.

Reply:
You are right, I will change it